### PR TITLE
[KEYCLOAK-8133] - Can't Sucessfully inject a custom KeycloakSpringBootConfigResolver in the Keycloak Spring Boot Security Adapter

### PR DESCRIPTION
**Pull request by pedroigor**

### CODE REVIEW

File: adapters/oidc/spring-boot-adapter-core/src/main/java/org/keycloak/adapters/springboot/KeycloakSpringBootConfigResolver.java [hunk 1]
```diff
@@ -17,29 +17,39 @@
...
-
     private static AdapterConfig adapterConfig;
+    private static KeycloakConfigResolver delegateConfigResolver;
+
+    private KeycloakDeployment keycloakDeployment;
 
     @Override
     public KeycloakDeployment resolve(OIDCHttpFacade.Request request) {
-        if (keycloakDeployment != null) {
+        if (delegateConfigResolver == null) {
```
[the remark this answers is not in the document]
Comment (pedroigor) [reply adapters/oidc/spring-boot-adapter-core/src/main/java/org/keycloak/adapters/springboot/KeycloakSpringBootConfigResolver.java]: @sebastienblanc Is there a JIRA for the `keycloak.json` issue?

[the remark this answers is not in the document]
Comment (sebastienblanc) [reply adapters/oidc/spring-boot-adapter-core/src/main/java/org/keycloak/adapters/springboot/KeycloakSpringBootConfigResolver.java]: @pedroigor I think and it was you that opened it ;) let me check

[the remark this answers is not in the document]
Comment (sebastienblanc) [reply adapters/oidc/spring-boot-adapter-core/src/main/java/org/keycloak/adapters/springboot/KeycloakSpringBootConfigResolver.java]: Can't find it , must been have been closed :/

[the remark this answers is not in the document]
Comment (pedroigor) [reply adapters/oidc/spring-boot-adapter-core/src/main/java/org/keycloak/adapters/springboot/KeycloakSpringBootConfigResolver.java]: @sebastienblanc, I would then merge this one as it is and work with `keycloak.json` issue in the future. Do we have a deal ?

**Review by sebastienblanc — APPROVED** [review]

**Pull request merged**